bundles/af-core/src/test/resources/form/termsandconditions/test-content.json

```diff
@@ -47,7 +47,7 @@
     "showApprovalOption": true,
     "jcr:title": "Terms And Conditions",
     "fd:tnc": true,
-    "showAsPopup": true,
```

Comment:
This is backward incompatible change, for now, let's keep the property in content structure change (ie) showAsPopup so that existing form continue to work

Reply:
Done